contracts/core/src/xchain/OmniPortalStorage.sol

```diff
@@ -46,7 +46,7 @@ abstract contract OmniPortalStorage is IOmniPortal, IOmniPortalAdmin {
     uint64 public omniChainId;
 
     /**
-     * @notice Virtual chain ID used in xmsgs from Omni's consensus chain
+     * @notice Chain ID of Omni's consensus chain
```

Comment:
Maybe we can leave "virtual"? or something like it

the "virtual" wording was here because it's not actually the chain id of the consensus chain. consensus chain is a cosmos chain, and does not have an evm chain id. We use a uint representation of the Omni's consensus chain  id for consensus xblocks.